### ISSUE TITLE
Update exercises

### ISSUE THREAD

**Issue by siwelwerd** [StevenClontz/tbil-la#126]
The exercises need to be updated to reflect changes discussed in #123 . These will be developed through MasterIt CoCalc, then copied in here when ready.

**Comment by siwelwerd** [comment]
Templates and code for all exercises have been written. They may not match the upstream MasterIt code yet; once they do, then we can pull them over here.

**Comment by siwelwerd** [comment]
@StevenClontz Is this moot now? Should we delete exercises from this repository?

**Comment by StevenClontz** [comment]
Yeah I'd move this discussion of exercises to https://github.com/TeamBasedInquiryLearning/checkit-tbil-la

Are there build scripts in this repo that need generated exercises? I *think* we're good with just a sample-exercises doc for now. (Although eventually checkit should support generating full `<solution>`s and could build that kind of file as well.)

**Comment by siwelwerd** [comment]
I think the exercises just generate the exercise library. The exercises here haven't been updated though, so they are behind what is in checkit-tbil-la, which is why I am thinking to just delete them.
